aiida/orm/nodes/data/code.py

```diff
@@ -131,21 +152,14 @@ def relabel(self, new_label, raise_error=True):
         if new_label.endswith(suffix):
             new_label = new_label[:-len(suffix)]
 
-        if '@' in new_label:
-            msg = "Code labels must not contain the '@' symbol"
-            if raise_error:
-                raise InputValidationError(msg)
-            else:
-                return InputValidationError(msg)
-
         self.label = new_label
 
     def get_description(self):
         """Return a string description of this Code instance.
 
```

Comment:
While you're at it, feel free to add a similar deprecation warning for `get_description` - one should use the `.description` property instead.